### PR TITLE
Ensure Government#edit page handles blank start date gracefully & design updates

### PR DESCRIPTION
**Pull request by davidgisbey**
## Description

At the moment, this is blowing up as a model on the validation expects start date to be present. This updates the validation for overlapping governments to only run when start_date is present.

It also fixes a bunch of issues with the code itself. Duplicate fields, values not being set in inputs correctly, label size etc. and implements the changes outlined by Nikin post his 1.8 audit. 

### Before 

image

image

### After

image

previous govt cannot be closed 

image

## Trello card 

https://trello.com/c/nSDC5uV7/191-revisit-edit-government-page

⚠️ This repo is Continuously Deployed: make sure you [follow the guidance](https://docs.publishing.service.gov.uk/manual/development-pipeline.html#merge-your-own-pull-request) ⚠️

Follow [these steps](https://guides.rubyonrails.org/upgrading_ruby_on_rails.html) if you are doing a Rails upgrade.

### CODE REVIEW

**Comment by nnagewad** [comment]
Looks good @davidgisbey 👍🏽

File: app/views/admin/governments/_form.html.erb [hunk 1]
```diff
@@ -73,7 +73,8 @@
         } %>
         <%= link_to("Cancel", admin_governments_path(), class: "govuk-link") %>
       </div>
-      <% if government.persisted? %>
+
+      <% if government.current? %>
         <%= link_to("Prepare to close this government", prepare_to_close_admin_government_path(government), class:
```
[the remark this answers is not in the document]
Comment (davidgisbey) [reply app/views/admin/governments/_form.html.erb:78]: good shout pushing one.

File: app/views/admin/governments/_form.html.erb [hunk 2]
```diff
@@ -15,60 +15,49 @@
...
           heading_size: "m",
-          date_hint: "For example, 01 August 2022",
-          date_heading: "End date",
+          date_hint: "For example, 01 August 2015",
+          date_heading: "Start date (required)",
           year: {
-            value: government.end_date.to_date,
-            start_year: 1800
+            value: government.start_date&.year,
+            start_year: 1800,
```
[the remark this answers is not in the document]
Comment (davidgisbey) [reply app/views/admin/governments/_form.html.erb:32]: yea sounds good to me 👍 Will sort it now

**Review by chriswhitehouse-tw — APPROVED** [review]
Looks good to me, David. 👍

**Pull request merged**